content/stratis-rootfs.md

```diff
@@ -119,4 +119,7 @@ filesystem will be included in stratisd 2.4.0.
 
 <!-- more -->
 
+### Corrections
+* 
```

Comment:
Looks like you didn't add the bit that was intended to go here?

Reply:
Whoops! Thanks.